starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -717,6 +693,44 @@ sub check_dither {
             }
         }
     }
+
+
+
+
+    # For eng obs, don't have OR to specify dither, so stop before doing vs-OR comparisons
+    if ( $self->{obsid} =~ /^\d*$/){
+	return if ($self->{obsid} >= $ER_MIN_OBSID);
+    }
+
+    # Get the OR value of dither and compare if available
+    my $bs_val = $dither->{state};
```

Comment:
It seems like there is inconsistent use of `$dither` and `$self->{cmd_*}`.  If possible, up top you should define those `$self->{cmd_*}` values and then forget about `$dither`.